src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/AddPropertyTagHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System.Threading.Tasks;
+
+namespace OrchardCore.DisplayManagement.TagHelpers;
+
+[HtmlTargetElement("add-property", Attributes = "name", TagStructure = TagStructure.NormalOrSelfClosing)]
```

Comment:
property is better IMHO

Reply:
`add-property` [was your idea](https://github.com/OrchardCMS/OrchardCore/pull/14867#discussion_r1485397119). Also it matches `add-alternate`, `add-wrapper`, etc.

Reply:
The name is meant to match with existing tags like `add-wrapper`, `add-alternate`, etc. It fits the pattern.

Reply:
Ok